cmd/cmd_synctarget.go

```diff
@@ -0,0 +1,156 @@
+package cmd
+
+import (
+	"context"
+	"github.com/lithictech/webhookdb-cli/appcontext"
+	"github.com/lithictech/webhookdb-cli/client"
+	"github.com/urfave/cli/v2"
+)
+
+var synctargetCmd = &cli.Command{
+	Name:  "sync-target",
+	Usage: "",
+	Subcommands: []*cli.Command{
+		{
+			Name:  "create",
+			Usage: "Create a sync target for the given organization.",
```

Comment:
Create a sync target for the specified integration. Data will be automatically synced from the integration's WebhookDB table into the database specified by the connection string. PostgresQL and SnowflakeDB databases are supported.